develop/codingguidelines/Attention/Minimum_DMA_version.md

```diff
@@ -8,3 +8,10 @@ Make sure the [MinimumRequiredVersion](xref:Protocol.Compliancies.MinimumRequire
 
 > [!TIP]
 > DIS performs a feature detection to detect whether the specified value for MinimumRequiredVersion is in accordance with the used features in the protocol.
+
+| Feature | Minimum DMA Version | Release Note |
+|----------|----------|----------|
+| Protocol clear and leave in FillArray    | 10.4.2.0-13691     | [Releas Note](https://intranet.skyline.be/DataMiner/Lists/Release%20Notes/DispForm2.aspx?ID=38153)     |
+
+> [!WARNING]
+> The table only contains known features that have a minimum DMA version impact. The list only starts from the [minimum supported DMA](https://docs.dataminer.services/user-guide/Reference/Software_support_life_cycles.html) version 10.2.
```

Comment:
As the minimum supported DataMiner version changes every year, it might be better to rephrase this to make sure it's future-proof. Note also that a warning is usually only used for things that can pretty much render your system useless. "[!IMPORTANT]" would be more appropriate here.

```suggestion
> [!IMPORTANT]
> The table only contains known features that have a minimum DMA version impact, starting from DataMiner 10.2.
```